Dependencies/napi/napi-extensions/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_library(napi_extensions INTERFACE)
+target_include_directories(napi_extensions INTERFACE "include")
```

Comment:
hmm, i wasn't thinking you would add a new cmake for this, but i guess it's fine.